### ISSUE TITLE
BUG - sev-2? - All - Header text missing with SR staying on (on category, feature and child screens)

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8310]
## What happened?

The screen reader + header functionality added in #5010 during 'new nav' implementation, is now broken.

What should happen/previously happened: if you had a screen reader on (no matter when you turned it on), the title of the page would replace 'VA' in the header, and the screen reader would read out the title (ex: Appointments)

What is currently happening: if you turn on a screen reader, the heading on the page you are on will move up to replace 'VA' and be read out correctly by the screen reader. On all other pages, the screen reader will announce the correct title, seemingly on an empty space.

Screenshot:
<img width="359" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzgzQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--dc460d26550455c50c72b0e4bee0ff560637d8df/image.png" alt="image.png" />

[Demonstrative video.](https://drive.google.com/file/d/1gOrO7WWTnXWX5hEV0yeSxaMc6TEoCoks/view?usp=drive_link) of how it's broken when I have SR on 'before' the page, but then fixes itself if I activate SR on the given page.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: High 

- 2 - High

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
Updated the title to make the problem easier to understand.

**Comment by TKDickson** [comment]
Wanted to just double check that this is working in develop, and I did a spotcheck on iOS that yes it is. Closing.
